view/frontend/web/js/view/payment/stash.js

```diff
@@ -120,7 +120,7 @@ define([
          */
         _getStash: function (stash_id) {                    
             var postData ={};
-            var url = "https://stash-sandbox.gointerpay.net/"+stash_id;
+            var url = window.checkoutConfig.reach.api_url + stash_id;
```

Comment:
see previous comment around changing api_url to stash_url or stash_api_url. I know in context it can be assumed that this is a stash url but it would still be preferable to be as explicit as possible.

Reply:
I second Lizzy. Thanks.

Reply:
Variable name has been changed.